docs/ops/config.md

```diff
@@ -77,6 +77,10 @@ without explicit scheme definition, such as `/user/USERNAME/in.txt`, is going to
 - `classloader.resolve-order`: Whether Flink should use a child-first `ClassLoader` when loading
 user-code classes or a parent-first `ClassLoader`. Can be one of `parent-first` or `child-first`. (default: `child-first`)
 
+- `classloader.always-parent-first-patterns`: A (comma-separated) list of patterns that specifies which classes should always be resolved
```

Comment:
are these prefixes; does it support regex?

Reply:
simple prefix, changing doc 👍